src/mpid/ch4/shm/gpu/gpu_control.c

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) by Argonne National Laboratory.
+ *     See COPYRIGHT in top-level directory.
+ */
+
+#include "mpidimpl.h"
+#include "gpu_pre.h"
+#include "gpu_impl.h"
+#include "gpu_recv.h"
+#include "gpu_control.h"
+
+int MPIDI_GPU_ctrl_send_lmt_recv_fin_cb(MPIDI_SHM_ctrl_hdr_t * ctrl_hdr)
+{
+    int mpi_errno = MPI_SUCCESS;
+    MPIR_FUNC_VERBOSE_STATE_DECL(MPID_STATE_GPU_CTRL_SEND_LMT_RECV_FIN_CB);
+    MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_GPU_CTRL_SEND_LMT_RECV_FIN_CB);
+
+    MPIR_FUNC_VERBOSE_EXIT(MPID_STATE_GPU_CTRL_SEND_LMT_RECV_FIN_CB);
+    return mpi_errno;
+}
```

Comment:
Why do we need a completely different protocol, instead of simply adding the IPC handle to the already existing LMT?

Reply:
Ok, that's fair. The protocol should be pretty simple.